openedx/features/content_tagging/rest_api/v1/tests/test_views.py

```diff
@@ -0,0 +1,180 @@
+"""
+Tests tagging rest api views
+"""
+
+import ddt
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from openedx.features.content_tagging.models import Taxonomy
+
+User = get_user_model()
+
+
+@ddt.ddt
+class TestTaxonomyViewSet(APITestCase):
+    def setUp(self):
+        super().setUp()
+
+        self.user = User.objects.create(
+            username="user",
+            email="user@example.com",
+        )
+
+        self.staff = User.objects.create(
+            username="staff",
+            email="staff@example.com",
+            is_staff=True,
+        )
```

Comment:
The only thing that we need to test for content_tagging vs oel_tagging are the "org" membership allowance for normal users, so we need to see the view permission checks here done with the various org membership options.

cf [test_rules.py](https://github.com/openedx/edx-platform/pull/32661/files#diff-67e5795cdd1a25095de77990b0e78015b2d49761e60600c5fca1ecee63b191d1R1) -- I recommend splitting out the `setUp` method in TestRulesTaxonomy into a Mixin you can use here, so you can borrow the user- and org- creation logic there.

But we'll need to see the view permissions checks done with those users, and with ENABLE_CREATOR_GROUP [turned on](https://github.com/openedx/edx-platform/pull/32661/files#diff-67e5795cdd1a25095de77990b0e78015b2d49761e60600c5fca1ecee63b191d1R19) and [turned off](https://github.com/openedx/edx-platform/pull/32661/files#diff-67e5795cdd1a25095de77990b0e78015b2d49761e60600c5fca1ecee63b191d1R322).

And please apply similar suggestions to your view tests here as I requested for https://github.com/open-craft/openedx-learning/pull/4/files/a0fedd36bbe06ef0a4c00466c8c9570f3c35616d#diff-2f11bfdf39bf0bee3b4778c0d84478fa6afd170e57245f804c77b2e9b4d865a4.

Reply:
Understood! This PR is still a WIP: I must create the tests/view related to the Org permissions.
Today I worked on the permissions here.